harvard-university-of-technology-sydney.csl

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="sort-only" version="1.0" page-range-format="minimal" default-locale="en-GB">
+  <info>
+    <title>Harvard - University of Technology Sydney</title>
+    <title-short>Harvard UTS</title-short>
+    <id>http://www.zotero.org/styles/harvard-university-of-technology-sydney</id>
+    <link href="http://www.zotero.org/styles/harvard-university-of-technology-sydney" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-swinburne-university-of-technology" rel="template"/>
+    <link href="https://www.lib.uts.edu.au/help/referencing/harvard-uts-referencing-guide" rel="documentation"/>
+    <author>
+      <name>Meijun Huang</name>
+      <email>meijun.huang@uts.edu.au</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>The Australian version of the Harvard author-date style</summary>
+    <updated>2016-03-15T03:09:51+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <group delimiter=" ">
+      <choose>
+        <if type="chapter paper-conference" match="any">
+          <text term="in"/>
+        </if>
+      </choose>
+      <names delimiter=", " variable="editor" suffix=",">
+        <name initialize-with="." and="symbol" delimiter-precedes-last="never"/>
+        <label form="short" prefix=" (" suffix=")"/>
+      </names>
+      <names delimiter=", " variable="translator" suffix=",">
+        <name initialize-with="." and="symbol" prefix="trans. "/>
+      </names>
+    </group>
+  </macro>
+  <macro name="anon">
+    <choose>
+      <if type="book figure graphic legal_case legislation manuscript pamphlet patent report thesis broadcast song post" match="any">
+        <text font-style="italic" variable="title" suffix=" "/>
+      </if>
+      <else-if type="bill" match="any">
+        <text variable="title" suffix=" "/>
+      </else-if>
+      <else-if type="webpage" match="any">
+        <choose>
+          <if variable="container-title" match="any">
+            <text variable="title" prefix="'" suffix="'"/>
```

Comment:
please use `<text variable="title" quotes="true"/>` instead. Straight quotes are ugly (and typographically incorrect).